rbi/annotations/shopify-money.rbi

```diff
@@ -0,0 +1,51 @@
+# typed: strong
+
+class Money
```

Comment:
`initialize` is slightly annoying to type because it will accept and try to coerce a lot of different types for both arguments, but I'll give it a try anyway.

The other methods are just an oversight, I'll add them too!